src/config/typeorm.config.ts

```diff
@@ -3,7 +3,7 @@ import { TypeOrmModuleOptions } from '@nestjs/typeorm';
 export const typeOrmConfig: TypeOrmModuleOptions = {
   type: 'postgres',
   host: 'localhost',
-  port: 5432,
+  port: 5555,
```

Comment:
can we move all these values to environment variables so everyone can change them however they want from their env config